application/dashboard/trello_service.py

```diff
@@ -35,25 +35,28 @@
     "5a686fb201a2b230f9de88cd": {"name": "Not being worked on", "id": "5a686fb201a2b230f9de88cd", "stage": "other"},
 }
 
-WORK_FLAGS = ("New measure", "Updated version")
-DEPARTMENT_FLAGS = (
-    "BEIS",
-    "CO",
-    "DCMS",
-    "DHSC",
-    "MHCLG",
-    "DEFRA",
-    "DfE",
-    "DfT",
-    "DH",
-    "DWP",
-    "HO",
-    "MoJ",
-    "ONS",
-    "RDU",
-    "MOD",
-    "HESA",
-)
+TYPE_OF_WORK_LABELS = {
+    "New measure": "New measure",
+    "Updated version": "Updated version",
+    "Updated version v3": "Updated version",
+}
+
+ORGANISATION_LABELS = {
```

Comment:
@TheDoubleK we actually have all these in the database. The `Organisation` table has an `abbreviations` column which is an array. Could we use that?

Reply:
Maybe, as long as the abbreviations there match the ones on the Trello board. I'll take a look. The list here is taken from the Trello board labels.

Reply:
They _should_ match. And if not, we can add the extra acronyms to the database.

Reply:
Hmm, I'm not sure about this... there are way more orgs in the database than the ones referenced on the Trello board (568 vs the 14 here), and the abbreviations in the DB are a list of possible abbreviations for each org, some of which look pretty fishy, and they're definitely not guaranteed to be unique.
And they are conceptually different things I think - this is a list of labels on a Trello board, which conceptually could be completely different from the Organisations we have in our DB.

Reply:
@TheDoubleK I dunno, they look similar enough to me, and it feels a bit pointless to maintain two separate lists of organisational acronyms (especially if one is in code)?

You can select them with SQL like this:
```sql
select organisation.* from organisation where 'MHCLG' = ANY (abbreviations);
```

For what it's worth, the list of abbreviations in the database comes a script I wrote to parse the GOV.UK organisations API (which includes abbreviations).

Reply:
Still looks fishy... like "The Ombudsman" is not an abbreviation for the specific ombudsman it relates to in our DB.
They look similar, but that's by chance.  The analysts could have chosen to use different abreviations on their Trello tickets and that should be OK.

Reply:
And we would still have to have the "Type of work" labels, so then we'd have two completely different ways of doing the same thing - i.e. match a label on a Trello card against a list of values we want to know about.

Reply:
And some of the abbreviatios don't (exactly) match our DB... eg. the Trelo board has "MoJ" but database has "MOJ"... yep I know this is trivial to fix, but I think analysts should be able to tag how they like on the Trello.